client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -1079,3 +1079,11 @@ body.is-section-signup.is-white-signup .is-onboarding-2023-pricing-grid .signup_
 		color: #55347d;
 	}
 }
+
+.plan-features-2023-grid__discount-promotion {
```

Comment:
Would it be possible to move this CSS to a styled component? We're hoping to move the grid over to styled-components so that it looks consistent across usages.